region/brinstar/blue.json

```diff
@@ -2257,6 +2226,66 @@
                   "note": "Starting an IBJ from spring ball with no other items is not very precise, it just takes a bit of an odd timing."
                 }
               ]
+            },
+            {
+              "id": 2,
+              "strats": [
+                {
+                  "name": "Leave with Runway",
+                  "notable": false,
+                  "requires": [],
+                  "exitCondition": {
+                    "leaveWithRunway": {
+                      "length": 1,
+                      "openEnd": 1
+                    }
+                  }
+                },      
+                {
+                  "name": "Leave Charged (with Gravity)",
```

Comment:
I think this and the next strat should be moved from `1->2`

Reply:
☝️

Reply:
Done.